assets/css/global.less

```diff
@@ -1175,14 +1175,17 @@ body .guides .guides-section-white {
 }
 
 #toc {
+  &.fixed {
```

Comment:
Why does this only apply when it's in `.fixed` mode? That class is only added while scrolling... Not when at the top or bottom of the page.

Reply:
That's exactly when it messes up. When the fixed class is added it starts ignoring the parent width for some reason.

Reply:
Ugh... CSS is hard. OK, ship it! 🎉